specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -608,6 +751,94 @@
           "type": "string"
         }
       }
+    },
+    "PutAliasRequest": {
+      "description": "The parameters required to create a new subscription.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PutAliasRequestProperties",
+          "description": "Put alias request properties."
+        }
+      }
+    },
+    "PutAliasRequestProperties": {
+      "description": "Put subscription properties.",
+      "required": [
+        "displayName",
+        "workloadType",
+        "billingScope"
+      ],
+      "properties": {
+        "displayName": {
+          "description": "The friendly name of the subscription.",
+          "type": "string"
+        },
+        "workloadType": {
+          "description": "The workload type of the subscription. It can be either Production or DevTest.",
+          "type": "string"
+        },
+        "billingScope": {
+          "description": "Determines whether subscription is fieldLed, partnerLed or LegacyEA",
+          "type": "string"
+        },
+        "subscriptionId": {
+          "description": "This parameter can be used to create alias for existing subscription Id",
+          "type": "string"
+        }
+      }
+    },
+    "PutAliasResponse": {
+      "description": "Subscription information (same as Microsoft.Resources/subscription).",
+      "properties": {
+        "id": {
+          "description": "Fully qualified ID for the subscription resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "name": {
+          "description": "Subscription ID.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type, Microsoft.Subscriptions.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/PutAliasResponseProperties",
+          "description": "Put subscription response properties."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "PutAliasResponseProperties": {
+      "description": "Put subscription creation result properties.",
+      "properties": {
+        "subscriptionId": {
+          "description": "Newly created subscription Id.",
```

Comment:
Mark this as readOnly: true for completeness

Reply:
Updated